### DISCUSSION TITLE
[错误报告]: 搜索：用中文名搜不出结果

### DISCUSSION THREAD

**Discussion by Whichbfj28** [jxxghp/MoviePilot#852] [讨论]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

1.3.2

### 问题类型

主程序运行问题

### 问题描述

搜索：长安三万里
![image](https://github.com/jxxghp/MoviePilot/assets/47915665/2f188681-7676-4cfb-bfff-86573f249558)

搜索：Chang An

![image](https://github.com/jxxghp/MoviePilot/assets/47915665/59979c96-7464-4ead-bff4-1077476c3b90)

问题：用中文名字无法搜索出结果

### 发生问题时系统日志和配置文件

__init__.py | 运行模块 search_medias 出错：TheMovieDbModule - 'NoneType' object has no attribute 'headers'

**Comment by baichenxu06** [comment]
纯粹是连接性问题

**Comment by Whichbfj28** [comment]
你说的连接性问题 指的是什么？连通吗。还是丢包。
连通的话是这样的
![image](https://github.com/jxxghp/MoviePilot/assets/47915665/93cfbe18-d080-4700-9e25-29a1c86aba06)
另外你那里是否可以测试下 是否正常。还是就是我这样而已

**Comment by baichenxu06** [comment]
> 你说的连接性问题 指的是什么？连通吗。还是丢包。 连通的话是这样的 ![image](https://user-images.githubusercontent.com/47915665/275163080-93cfbe18-d080-4700-9e25-29a1c86aba06.png) 另外你那里是否可以测试下 是否正常。还是就是我这样而已

能搜到，是不是挂代理了，代理设置的问题

**Comment by cng10001** [comment]
搜索功能几乎人人要用吧，出问题肯定要及时修的

https://github.com/jxxghp/MoviePilot/assets/115156545/a0b2320b-e8e8-4ca3-a798-1baf5eab9620

**Comment by Putarku** [comment]
> ### 确认
> * [x]  我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
> * [x]  我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
> * [x]  我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
> * [x]  我已经修改标题，将标题中的 描述 替换为我遇到的问题。
> 
> ### 当前程序版本
> 1.3.2
> 
> ### 问题类型
> 主程序运行问题
> 
> ### 问题描述
> 搜索：长安三万里 ![image](https://user-images.githubusercontent.com/47915665/275148495-2f188681-7676-4cfb-bfff-86573f249558.png)
> 
> 搜索：Chang An
> 
> ![image](https://user-images.githubusercontent.com/47915665/275148221-59979c96-7464-4ead-bff4-1077476c3b90.png)
> 
> 问题：用中文名字无法搜索出结果
> 
> ### 发生问题时系统日志和配置文件
> **init**.py | 运行模块 search_medias 出错：TheMovieDbModule - 'NoneType' object has no attribute 'headers'

我这边测试了下没有问题，推荐手动去代理那边把tmdb手动加入代理规则，不然有时候就测试通，但实际老是出问题
![image](https://github.com/jxxghp/MoviePilot/assets/66679443/652f623a-4c0a-4f2c-9e9b-19392279caac)

**Reply by Whichbfj28** [reply]
确认 这不是代理的问题
TMDB_API_DOMAIN= 这地方不要设置为api.tmdb.org 不然就会出现题中情况
修改为TMDB_API_DOMAIN=api.themoviedb.org即可。
个人猜测是因为api.tmdb.org使用的是跳转方式。而不是反代方式所导致的
